cmds/exp/pxeserver/main.go

```diff
@@ -142,6 +151,7 @@ func main() {
 			mac:          maca,
 			self:         net.ParseIP(*selfIP),
 			yourIP:       yourIP,
+			bcast:        yourIP.Mask(yourNet.Mask),
```

Comment:
Is this your subnet?

Reply:
it is, but it seems to keep the kernel happy, so I'm good with it.

Reply:
at some point possibly subnet is a better name, but until we've got more testing, i'm just not sure.